scripts/e2e/e2e.sh

```diff
@@ -86,6 +84,7 @@ get_bootstrap_vm() {
       echo "bootstrap vm ip is empty"
       exit 1
    fi
+   vm_created="true"
```

Comment:
Hi @codenrhoden,

This is a global variable since it is consumed in `exit_cleanup_on_err` and so should be upper case and defined earlier with a comment:

```shell
# VM_CREATED is set to "true" when the VM has been successfully created.
VM_CREATED=
```

Then it should be assigned, well, anything when created:

```shell
   VM_CREATED=1
```